python/dgl/graph.py

```diff
@@ -3045,7 +3045,7 @@ def edge_subgraph(self, edges, preserve_nodes=False):
         sgi = self._graph.edge_subgraph(induced_edges, preserve_nodes=preserve_nodes)
         return subgraph.DGLSubGraph(self, sgi)
 
-    def adjacency_matrix_scipy(self, transpose=False, fmt='csr'):
+    def adjacency_matrix_scipy(self, transpose=False, fmt='csr', return_edge_ids=True):
```

Comment:
I agree, and I think it should defaults to False.

Reply:
Do we have existing code that depend on this behavior?

Reply:
My transformer depend on this behavior and I would refactor the code.
Some unittests may depend on this function but I guess they expect this function to return all 1s.